src/app/core/modules/course.ts

```diff
@@ -0,0 +1,7 @@
+export interface ICourse {
+  id: number;
+  title: string;
+  creation: string;
+  duration: string;
+  description: string;
+}
```

Comment:
Please, rename the directory from 'modules' to 'models'.

Reply:
done